src/Recurr/Transformer/ArrayTransformer.php

```diff
@@ -89,7 +89,7 @@ public function transform(Rule $rule, ConstraintInterface $constraint = null, $c
         $end   = $rule->getEndDate();
         $until = $rule->getUntil();
 
-        if (null === $start) {
+        if (null === $start || !$start instanceof \DateTime) {
```

Comment:
This will have to be `\DateTimeInterface` as per https://github.com/passeride/recurr/blob/b8e2b86d3ce2b03ab5ec68e71472bad75105b9e7/src/Recurr/Rule.php#L593

Reply:
@passeride Can you make the change to `\DateTimeInterface`? I'll merge in after.

Reply:
If $start is a string, what's in the string? Is it a date as a string? Should it be converted to a DateTime in that case instead of taking `'now'` as the date? Not sure what the use case is but this fix sounds dodgy to me.

Reply:
Looking at this again, I agree. I actually am thinking I should throw an exception there if `$start` isn't null and isn't an instance of `\DateTimeInterface`